packages/trader/src/AppV2/Components/TradeTypeList/trade-type-list.tsx

```diff
@@ -64,7 +64,7 @@ const TradeTypeList: React.FC<TTradeTypeListProps> = ({
                                 className='trade-type-list-category-header-button'
                                 onClick={onAction}
                             >
-                                {category.button_title || <Localize i18n_default_text='Customize' />}
+                                {category.button_title || <Localize i18n_default_text='Customise' />}
```

Comment:
This spelling issue was also fixed here in DTRA-1887 (in progress qa)

Reply:
its ok. i just merge master when that card is merged ?

Reply:
Up to you